sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/HttpPipelineHelper.cs

```diff
@@ -23,15 +24,24 @@ internal static class HttpPipelineHelper
 
         internal static int GetItemsAccepted(HttpMessage message)
         {
-            return GetTrackResponse(message).ItemsAccepted.GetValueOrDefault();
+            return TryGetTrackResponse(message, out var trackResponse)
+                ? trackResponse.ItemsAccepted.GetValueOrDefault()
+                : default;
         }
 
-        internal static TrackResponse GetTrackResponse(HttpMessage message)
```

Comment:
A few methods in this class needed to be change to a TryGet pattern because the dependent libraries now specify that they can return null.